main.go

```diff
@@ -1,5 +1,59 @@
 package main
 
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/config"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/forman"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/reconciler"
+	"go.uber.org/zap"
+)
 
 func main() {
+	//logger := logr.Logger{}
+	ctx := context.Background()
+	configFilePath := os.Getenv("CONFIG_FILE_PATH")
+	if configFilePath == "" {
+		log.Fatal("CONFIG_FILE_PATH environment variable not set")
+	}
+	// parse configuration from yaml file
+	conf, err := config.LoadAndValidateConfig(configFilePath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	zapConfig := zap.Config{
+		Encoding:         "json",
+		EncoderConfig:    zap.NewProductionEncoderConfig(),
+		OutputPaths:      []string{"stdout"},
+		ErrorOutputPaths: []string{"stderr"},
+	}
+	logger, err := zapConfig.Build()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	dbInstance, err := db.NewClient(ctx, &conf.DB)
```

Comment:
This is not an instance, it is more a client. Lets reflect that in the name
```suggestion
	dbClient, err := db.NewClient(ctx, &conf.DB)
```